configure.ac

```diff
@@ -472,8 +473,10 @@ if test -n "$CHECK_SOS"; then
 fi
 if test -n "$SOS_LIB64DIR"; then
 	soslibdir="$SOS_LIB64DIR"
+	plugins_rpath="$plugins_rpath:$soslibdir"
```

Comment:
nit: Can't we just move this line to outside the if since it's the same line of code on both sides of the if/else.

Reply:
I was in a hurry; it should have been the 4 line change now provided instead of two. The revision stops append of empty values to plugins_rpath.